react/react1/week2/todo-tracker/src/components/WatchCount.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import { useEffect } from "react";
+function WatchCount() {
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    let timer = setTimeout(() => {
```

Comment:
timer variable is not being changed anywhere. Should probably be defined as a const then?